kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVFileReader.java

```diff
@@ -228,6 +228,9 @@ public Void getCurrentValue() throws IOException, InterruptedException {
 
     @Override
     public float getProgress() throws IOException, InterruptedException {
+      if (size == 0) {
```

Comment:
Should this also remove the "avoid divide by zero errors" code path in initialize?